### DISCUSSION TITLE
How does libcurl websockets API work with curl_multi API?

### DISCUSSION THREAD

**Discussion by perkfly** [curl/curl#11867] [Q&A]
I am trying to build an application that read and write asynchronously with websockets. Previously, I was using `curl_multi_socket_action` and `curl_multi_info_read` for http requests. How does `curl_ws_*` APIs fit in there?

**Comment by dfandrich** [comment]
Have you read the docs? There's an overview at
https://github.com/curl/curl/blob/master/docs/WEBSOCKET.md
and man pages for the new calls, such as
https://curl.se/libcurl/c/curl_ws_send.html  There's also a nearly hour-long
video on the topic on https://curl.se/docs/videos/

**Reply by perkfly** [reply]
Thanks for the video link, I guess `curl_ws_poll` is the missing API that I hope to exist. Are there any plans on implementing it?
